topo/node/cisco/cisco.go

```diff
@@ -405,6 +408,18 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 		if pb.Config.Image == "" {
 			pb.Config.Image = "xrd:latest"
 		}
+		if pb.HostConstraints == nil {
```

Comment:
also please confirm which constraint applies only to xrd or only to 8000e if that is the case

Reply:
Addressed. Constraints applicable per model:
xrd : fs.inotify.max_user_instances
8000e : kernel.pid_max